GoodreadsEDA/mk_bklst.py

```diff
@@ -33,28 +39,29 @@ def write_to_csv(out_file, auth_bks, auth_name, append=True):
     if not append:
         writer.writerow(["Author_Name", "Book_Title", "Goodreads_Link"])
     # Write all of the boot data.
-    for index, row in enumerate(auth_bks):
+    for index in range(len(auth_bks)):
         info = [auth_name,
                 auth_bks[index].get_text()[1:-1],
                 f"https://www.goodreads.com{auth_bks[index].attrs['href']}",
-                ]
+               ]
         writer.writerow(info)
 
 
 def main():
     for auth_url in auth_urls:
         # The author's name is listed at the end of each url, after the last '.'
         auth_name = auth_url[auth_url.rfind(".") + 1:]
-        print(auth_name)
+        print(f"Scraping books by {auth_name}")
         auth_bks = get_auth_bks(auth_url)
 
         # Create booksnlinks.csv if it doesn't exist already.
-        if not os.path.exists(f"{os.getcwd()}/csv_files/booksnlinks.csv"):
-            with open(f"{os.getcwd()}/csv_files/booksnlinks.csv", "w") as out_file:
-                write_to_csv(out_file, auth_bks, auth_name, append=False)
-        else:
+        if os.path.exists(f"{os.getcwd()}/csv_files/booksnlinks.csv"):
```

Comment:
I changed this because it ended up becoming a double negative in the `write_to_csv` method with the `append` variable. Generally, the goal of `if` statements is to have as few conditionals as possible, which makes it quicker to grasp the logic of